docs/adr/0001-compatibility-with-different-play-version.md

```diff
@@ -0,0 +1,56 @@
+# Making the sbt-accessibility-plugin Compatible with various Play Framework versions
+
+* Status: proposed
+* Deciders: PlatUI
+* Date: 29 Jan 2024
+
+Technical Story: [PLATUI-2755](https://jira.tools.tax.service.gov.uk/browse/PLATUI-2755)
+
+## Context and Problem Statement
+
+How can we make the future versions of the sbt-accessibility-plugin compatible with different versions of the Play Framework? We need a solution that balances ease of implementation, maintainability, and compatibility.
+
+## Decision Drivers
+
+* Compatibility with multiple versions of Play Framework
+* Maintainability of the plugin
+* Ease of implementation
+* Future-proofing against new Play Framework versions
+
+## Considered Options
+
+* Option 1: Build a specific version of the `sbt-accessibility-linter` plugin for each Play Framework version.
+* Option 2: Use reflection in the `sbt-accessibility-linter` to determine the consuming project's Play version and dynamically include the correct `scalatest-accessibility-linter` library.
+* Option 3: Employ macros to determine the Play version at compile time and include the appropriate `scalatest-accessibility-linter` library.
+* Option 4: Decouple the `scalatest-accessibility-linter` library from the `sbt-accessibility-linter` plugin.
+
+## Decision Outcome
+
+Chosen option: ???
+
+### Positive Consequences
+
+### Negative Consequences
+
+## Pros and Cons of the Options
+
+### Option 1: Specific Plugin Version for Each Play Version
+
+* Good, because it avoids the need for a multi-project build.
+* Bad, because it requires separate builds for each new Play version, increasing maintenance.
+
+### Option 2: Reflection to Determine Play Version
+
+* Good, because it eliminates the need for different plugin versions.
+* Good, because it's relatively easy to implement.
```

Comment:
```suggestion
* Good, because it's relatively easy to implement. We already have an implementation with some test coverage contributed from @wolfendale
```